File: ios/TasteByteERP/Core/Models/Attendance.swift
import Foundation

struct Attendance: Codable, Identifiable {
    let id: String
    let employeeId: String
    let date: Date
    let clockIn: Date?
    let clockOut: Date?
    let status: String
    let hoursWorked: Double?
    let notes: String?
}

struct ClockInResponse: Codable {
    let id: String
    let employeeId: String
    let clockIn: Date
    let status: String
}

struct ClockOutResponse: Codable {
    let id: String
    let employeeId: String
    let clockIn: Date
    let clockOut: Date
    let hoursWorked: Double
    let status: String
}

File: android/app/src/main/java/com/tastebyte/erp/models/Attendance.kt
package com.tastebyte.erp.models

import com.google.gson.annotations.SerializedName

data class Attendance(
    @SerializedName("id") val id: String,
    @SerializedName("employee_id") val employeeId: String,
    @SerializedName("date") val date: String,
    @SerializedName("clock_in") val clockIn: String?,
    @SerializedName("clock_out") val clockOut: String?,
    @SerializedName("status") val status: String,
    @SerializedName("hours_worked") val hoursWorked: Double?,
    @SerializedName("notes") val notes: String?,
    @SerializedName("created_at") val createdAt: String
)
